erpnext/selling/page/point_of_sale/point_of_sale.js

```diff
@@ -1526,9 +1526,41 @@ class Payment {
 		var me = this;
 
 		this.dialog.set_primary_action(__("Submit"), function() {
-			me.dialog.hide();
-			me.events.submit_form();
-		});
+			var company = me.frm.doc.company
```

Comment:
Why variable company, if the size of the fieldname is big then use the variable otherwise use me.frm.doc.company

Reply:
Fixed